src/DGtal/io/Display3DFactory.ih

```diff
@@ -1504,7 +1504,7 @@ void DGtal::Display3DFactory<Space,KSpace>::drawAsGrid( Display & display,
 {
   DGtal::Color fillColorSave = display.getFillColor();
   ASSERT(dim == 3);
-  DGtal::Z3i::RealPoint rp = display.embed( p );
+  DGtal::Z3i::RealPoint rp = display.embed(  DGtal::Z3i::Point(p, functors::Round<>() )  );
```

Comment:
If I understand this part (even before your modification): given a point in `Z^n` or `R^n`, we project it in `Z^n` and then embed it (back) in `R^n` (since the `Display3D` embedder is the canonical one). We specifically want to align the point with `Z^n` or can we simply write:
```C++
DGtal::Z3i::RealPoint rp(p);
```
?

Reply:
I was a bit puzzled by the method interface.. Let me check

Reply:
And about this, is this also OK?

Reply:
ha.. let me check..

Reply:
Yeah I agree with you.. `DGtal::Z3i::RealPoint rp(p);` would do the job...

Reply:
Anyway, we can postpone this question :wink: